starburst-presto/README.MD

```diff
@@ -18,6 +18,6 @@ You can find more information about using initialization actions with Dataproc i
 ## Important notes
 * This script must be updated based on which Presto version you wish to install
 * You may need to adjust the memory settings in `jvm.config` based on your needs
-* Presto is set to use HTTP port `8080` by default
+* Presto is set to use HTTP port `8080` by default, but can be changed using `--metadata dataproc-presto-port=8060`
```

Comment:
@medb what's the convention here? Should this be prefixed with `dataproc-`? `--metadata presto-port=8060` could be sufficient to me.

Reply:
Done.
No need for `dataproc-` prefix.